app/util/path.js

```diff
@@ -146,16 +146,16 @@ export const getStopRoutePath = searchObj => {
       path = `/${PREFIX_TERMINALS}/`;
       id = id.replace('GTFS:', '').replace(':', '%3A');
       break;
-    case 'bikeRentalStation':
+    case 'vehicleRentalStation':
       path = `/${PREFIX_BIKESTATIONS}/`;
       id = searchObj.properties.labelId;
       break;
     case 'bikestation':
       path = `/${PREFIX_BIKESTATIONS}/`;
       id = searchObj.properties.id;
       break;
-    case 'favouriteBikestation':
-    case 'favouriteBikeRentalStation':
```

Comment:
There are still some references to this in code. Not sure if the same or different thing.